### PR TITLE
[FIX] core: post_load default value should be same as *_init_hook

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
-Step to reproduce: add a post_load method in the init of any module, specify in the manifest like : 'post_load': 'post_load'. Run test_manifests of the test_lint module and we will get warning

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by duong77476-viindoo** [comment]
Dear @HydrionBurst Can you take a look

**Comment by HydrionBurst** [comment]
summon @Julien00859

**Comment by duong77476-viindoo** [comment]
Dear Mr @Julien00859 Can you reply to the summon call of Mr @HydrionBurst please :pleading_face:

**Comment by duong77476-viindoo** [comment]
Hi @Julien00859 How are you sir ? Can you take a look ?

**Comment by duong77476-viindoo** [comment]
@Julien00859 Hi sir, can you have a look ?

**Comment by Julien00859** [comment]
Hello @duong77476-viindoo sorry I was on holiday last week. I'm looking at it now :+1:

**Comment by Julien00859** [comment]
Thank you for your contribution, do you mind re-targeting your fix on 15.0? (you can keep this branch/pr)
So we fix all versions at once! :D

**Comment by duong77476-viindoo** [comment]
> Thank you for your contribution, do you mind re-targeting your fix on 15.0? (you can keep this branch/pr, just reset it on 15.0 and change the target) So we fix all versions at once! :D

Done sir, but v15.0 seem doesn't have the test to check, so probably when r+ someone will need to handle conflict and fix i guest

File: odoo/modules/module.py [hunk 1]
```diff
@@ -341,7 +341,7 @@ def load_information_from_description_file(module, mod_path=None):
             'description': '',
             'icon': get_module_icon(module),
             'installable': True,
-            'post_load': None,
+            'post_load': '',
```
Comment (duong77476-viindoo) [inline odoo/modules/module.py:344]: I search across v15 , i notice this only :v

**Comment by Julien00859** [comment]
> Done sir, but v15.0 seem doesn't have the test to check, so probably when r+ someone will need to handle conflict and fix i guest

Yes, I'll handle the forward-port, don't worry :)

@robodoo r+

**Pull request closed without merging**